Dockerfile

```diff
@@ -24,14 +24,10 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN mkdir -p ~/.ssh \
     && ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
 
-WORKDIR /code
+COPY requirements.txt .
```

Comment:
Not sure what happened here, but a few files got lost. Also, copying to `.` in Dockerfiles is strange; it is less readable for the user, and it might end up somewhere unexpected. What was your intention with this change?